src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Truck truck = new Truck();
```

Comment:
```suggestion
        Machine truck = new Truck();
```
same everywhere - we should better use more abstract type of variable